src/Admin/Model/UserAdmin.php

```diff
@@ -87,9 +81,8 @@ protected function configureFormFields(FormMapper $form): void
             ->end()
             ->tab('Security')
                 ->with('Status', ['class' => 'col-md-4'])->end()
-                ->with('Groups', ['class' => 'col-md-4'])->end()
                 ->with('Keys', ['class' => 'col-md-4'])->end()
-                ->with('Roles', ['class' => 'col-md-12'])->end()
+                ->with('Roles', ['class' => 'col-md-16'])->end()
```

Comment:
This is wrong. 12 is the max. You should instead change the col-md-4 to col-md-6 for status and keys.